Doc/Tutorial/chapter_seqio.tex

```diff
@@ -324,7 +324,7 @@ \section{Parsing sequences from compressed files}
 the compressed file for reading - which gives us a handle object:
 
 %TODO: Can we make the doctest code Python version specific?
-%This doctest fails on Python 2.7 due to http://bugs.python.org/issue30012
+%This doctest fails on Python 2.7 due to https://bugs.python.org/issue30012 
```

Comment:
It looks like you've added a trailing space here (at least, that's what it looks like in GitHub)